Fix/doc 4273 (#80)
exoplatform/exo-documentation: docs/Manage-Space.rst

1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.

> 2. A popup is displayed containing all informations related to the space creation.

Reviewer: The word information is uncountable, it does not take "s".

> 2. A popup is displayed containing all informations related to the space creation.

Reviewer: ```suggestion
2. A popup is displayed containing all information related to the space creation.
```

|image23|

> 3. The space creator have to enter a space name (required and from 3 to 200 characters) 

Reviewer 2: ```suggestion
3. The space creator must enter a space name (3 to 200 characters) 
```

>    and may enter an optional description (max 255 characters).

Reviewer: I think that we should precise the minimum characters which is 0 for description i.e. it not mandatory to fill it.

Author: While we have "optional" description, it means that the minimum is 0.

>    and description (from 0 to 255 characters).

Reviewer 2: ```suggestion
   and may enter an optional description (max 255 characters).
```

.. note:: Special characters are not allowed on space names. Only the character **&** is allowed.
           
           |image75|

.. _space-templates:

> 4. Space templates pulldown will presents all predefined templates that the space creator can use. 

Reviewer 2: ```suggestion
4. A pulldown lists all predefined templates that the space creator can use. 
```

> A description of every template in the pulldown is ensured and displayed just below the templates selector so that the space creator can differentiate between all templates.

Reviewer 2: ```suggestion
A description of each template is displayed below the selectors.
```

> .. note:: Every template have its specific visibility, regitration criteria and applications.

Reviewer 2: ```suggestion
.. note:: Each template defines its default visibility, registration criteria and list of applications.
```

.. _access-level-step:

> 4. The space creator may override the predefined visibility and registration options of the space.

Reviewer: ```suggestion
5. The space creator may override the predefined visibility and registration options of the space.
```

> 4. The space creator can modify the predefined visibility and registration of it space.

Reviewer 2: ```suggestion
4. The space creator may override the predefined visibility and registration options of the space.
```

.. _About-space-visibility:

> -  **Visibility**: By default, the Visibility value is set to be the one related to the template.

Reviewer 2: ```suggestion
-  **Visibility**: By default, the Visibility value is set to be the one predefined in the template.
```

	-  Visible: The space is always visible in the public spaces list.

	-  Hidden: The space is not visible in the public spaces list.

> -  **Registration**: By default, the Registration value is set to the one related to the template.

Reviewer 2: ```suggestion
-  **Registration**: By default, the Registration value is set to the one predefined in the template.
```

   -  Close: The user cannot request for joining, but only the space
      manager can invite him.

> 5. In the **Users ** field, the space creator can add users or space members to the space to create just by adding them in the selector.

Reviewer 2: ```suggestion
5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
```

> 5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.

Reviewer: ```suggestion
6. In the **Users** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
```

> All users to add in the space are displayed by a suggestor to facilitate their add.

Reviewer 2: ```suggestion
Type-ahead suggestions allow to facilitate selection users and groups to invite.
```

> .. note:: The space creator can display only spaces that he is a member on them.

Reviewer 2: ```suggestion
.. note:: Only spaces that the creator is member of can be selected.
```

> 6. Click Create Space button to finish adding your new space. The new space appears.

Reviewer: ```suggestion
7. Click Create Space button to finish adding your new space. The new space appears.
```

|image27|

.. _Space-templates:

**Space Templates**

Space templates concept is not directly exposed to users. From their perspective, they simply pick a type when creating a space.

> If the space creator choose a template from the pulldown, a description of the selected template is displayed below the pulldown:

Reviewer: ```suggestion
If the space creator chooses a template from the pulldown, a description of the selected template is displayed below the pulldown:
```

> If the space creator choose a template from the templates pulldown, a description of the selected template will be displayed below the pulldown:

Reviewer 2: ```suggestion
If the space creator choose a template from the pulldown, a description of the selected template is displayed below the pulldown:
```

   -  Community: A general purpose area for collaboration and communication of a digital community.
   -  Projects: A project space where members coordinate on tasks toward a predefined outcome.
   -  Communication : Interpersonal communication where a space groups are involved in exchange of ideas, skills and interests.
   -  Team: A central destination for members of a team.

>    .. note:: By default, in the create space popup we will have the Community template as default value.

Reviewer 2: ```suggestion
   .. note:: The default template for new spaces is:  Community

          Show More bar will appear at the page bottom. Click Show More
          to see more members.

.. note:: In the space settings, you can get an idea of the original space template used to create
         your space, by looking at the Template field 

> 		  the pulldown will be disabled.

Reviewer 2: ```suggestion
		  but you can't change it.
```

          Show More bar will appear at the page bottom. Click Show More
          to see more members.

> .. note:: In the space seeting, you can have an idea about the space template 

Reviewer 2: ```suggestion
.. note:: In the space settings, you can get an idea of the original space template used to create
```

>           for your space, the Space template pulldown will display the selected template and 

Reviewer 2: ```suggestion
         your space, by looking at the Template field 
```

> 		  the pulldown will be disabled.

Reviewer 2: but you can't change it.
